libraries/AP_Camera/AP_RunCam.h

```diff
@@ -98,6 +98,8 @@ class AP_RunCam
     void update();
     // Check whether arming is allowed
     bool pre_arm_check(char *failure_msg, const uint8_t failure_msg_len) const;
+    // whether or not we can arm
+    bool is_arming_prevented() const { return _in_menu > _menu_enter_level; }
```

Comment:
Probably want an 
```suggestion
    bool in_menus() const { return _in_menu > _menu_enter_level; }
```

and have is_arming_prevented call it.

Reply:
Thank you for your time.
Done